src/frontend/src/components/common/InputText/InputText.tsx

```diff
@@ -81,6 +81,14 @@ const InputText = ({
       inputRefCurrent.removeEventListener('keydown', keydownListener);
   }, [keydownListener]);
 
+  const checkTextArea = () => {
+    if (type === 'textArea') {
```

Comment:
can we create this as an enum in an interface?

export interface ItextType {
   INPUTTEXT : 'text',
  TEXTAREA: "textArea"
}
this way we can avoid spelling msitake when referencing textArea

Reply:
Except for the camel casing within the interface name, I agree with Anthony.